simulator/wdc-simulator.js

```diff
@@ -462,32 +459,40 @@
               }),
 
               PhaseTitle.element({ title: 'Phase 2: Data Gathering', isInProgress: dataGatheringStateInProgress }),
-              Button.element({ onClick: this.gatherData, disabled: isInProgress }, 'Run Gather Data'),
+              Button.element({ onClick: this.gatherData, disabled: isInProgress || isWDCUrlEmpty }, 'Run Gather Data'),
+              dataGatheringStateInProgress ? Button.element({ onClick: this.cancelCurrentPhase }, 'Cancel Gather Data Phase') : null,
+
               wdcCommandSim.canDoAnIncrementalRefresh()
                 ? Button.element({ onClick: this.incrementalRefresh, disabled: isInProgress }, 'Run Incremental Refresh')
                 : null
-
             ),
 
             Col.element({ md: 6, className: 'interactive-phase' },
               DOM.h2({}, 'Web Data Connector Properties'),
-              this.state.wdcUrlDisabled
-                ? Button.element({ onClick: this.unlockWdcProperties }, 'Unlock Properties')
-                : Button.element({ onClick: this.lockWdcProperties }, 'Lock Properties'),
-              Button.element({ onClick: this.clearWdcProperties }, 'Clear Properties'),
+              DOM.div({style: { marginBottom: '8px' }},
+                Button.element({ onClick: this.clearWdcProperties }, 'Clear Properties')
+              ),
 
               SimulatorProperties.element({
                 disabled: this.state.wdcUrlDisabled,
+                phaseInProgress: isInProgress,
```

Comment:
This goes with the comment from line 702. Instead of adding a new property, the line above should be changed to `disabled: isInProgress,`